lib/src/base.dart

```diff
@@ -113,6 +113,12 @@ class FlutterWebviewPlugin {
   /// Will trigger the [onDestroy] event
   Future close() => _channel.invokeMethod("close");
 
+  Future reload() => _channel.invokeMethod("reload");
+
```

Comment:
please comment functions and explain that it is only available on Android for now